webpack/app/upload.js

```diff
@@ -0,0 +1,131 @@
+import './upload.less';
+import './upload-icon.png';
+
+import 'get-image-data/jquery.getimagedata.js';
+import 'blueimp-tmpl/js/tmpl.js';
+import 'blueimp-load-image/js/index.js';
+import 'blueimp-canvas-to-blob/js/canvas-to-blob.js';
+import 'blueimp-file-upload/js/jquery.iframe-transport.js';
+import 'blueimp-file-upload/js/jquery.fileupload.js';
+import 'blueimp-file-upload/js/jquery.fileupload-process.js';
+import 'blueimp-file-upload/js/jquery.fileupload-image.js';
+import 'blueimp-file-upload/js/jquery.fileupload-audio.js';
+import 'blueimp-file-upload/js/jquery.fileupload-video.js';
+import 'blueimp-file-upload/js/jquery.fileupload-validate.js';
+import 'blueimp-file-upload/js/jquery.fileupload-ui.js';
+
+
+class Upload {
+  constructor() {
+    this.refresh();
+    this.bind_events();
+    if (this.$el.length > 0) {
+      this.start_upload();
+    }
+  }
+
+  refresh() {
+    this.$el = $('.fileupload');
+  }
+
+  bind_events() {
+    //overlay
+    $('#plone-contentmenu-factories #multiple-files').prepOverlay({
+      subtype: 'ajax',
+      config: {
+        onLoad: $.proxy(this.start_upload, this)
+      }
+    });
+    $(document).bind('drop dragover', (e) => {
+      // Prevent the default browser drop action:
+      e.preventDefault();
+    });
+    $(document).bind('drop', this.drop_image);
+  }
+
+  start_upload() {
+    // Overlay requires to refresh element object
+    this.refresh()
```

Comment:
You got a point.  But `jshint` should point `webpack` folder, what is outside the `.py` content.  We have two ways to fix it, review codeanalysis to change the js and css path, or add jshint with webpack.